Xamarin.Forms.Platform.iOS/Renderers/FrameRenderer.cs

```diff
@@ -23,24 +29,29 @@ protected override void OnElementChanged(ElementChangedEventArgs<Frame> e)
 
 			if (e.NewElement != null)
 			{
-				// Add the subviews to the actual view.
-				foreach (var item in NativeView.Subviews)
-				{
-					_actualView.AddSubview(item);
-				}
-
-				// Make sure the gestures still work on our subview
-				if (NativeView.GestureRecognizers != null)
+				if (_actualView == null)
 				{
-					foreach (var gesture in NativeView.GestureRecognizers)
-						_actualView.AddGestureRecognizer(gesture);
-				}
-				else if (_actualView.Subviews.Length == 0)
-				{
-					_actualView.UserInteractionEnabled = false;
+					_actualView = new UIView();
+					// Add the subviews to the actual view.
+					foreach (var item in NativeView.Subviews)
+					{
+						_actualView.AddSubview(item);
```

Comment:
@jfversluis 

When this is first hit nothing happens because the subviews are added by the VisualElementPackager

https://github.com/xamarin/Xamarin.Forms/blob/719fc7a604ff0cce8922d717c99bfb0fa17e35e0/Xamarin.Forms.Platform.iOS/VisualElementPackager.cs#L104

I don't think adding _actualView has really ever done anything TBH because subviews will always be empty on the first pass and then the VisualElementPackager will add it directly to the renderer later on. So, it appears that it does something but the subview is never actually added _actualView

Reply:
So the thing I _tried_ to do is basically have a bottom view which does the shadow and the top view that holds the actual content and is clipped etc. This so that the shadow isn't clipped when stuff like that happens.

The tricky thing is that some properties have to be only applied to the top view, like the clipping, but some to both, like translation.

I wasn't really aware of the packager thing. Like you say; it appeared to work. I got inspiration from PancakeView which also does it largely like this: https://github.com/sthewissen/Xamarin.Forms.PancakeView/blob/master/src/Xamarin.Forms.PancakeView.Multi/Platforms/iOS/PancakeViewRenderer.cs

I'll pull it down and do some quick tests on this